apis/resourcecontrollerv2/v1alpha1/resourceinstance_types.go

```diff
@@ -74,7 +71,7 @@ type ResourceInstanceObservation struct {
 
 	// When you create a new resource, a globally unique identifier (GUID) is assigned. This GUID is a unique internal
 	// identifier managed by the resource controller that corresponds to the instance.
-	GUID string `json:"guid,omitempty"`
+	Guid string `json:"guid,omitempty"`
```

Comment:
Our generator does uppercase a select set of acronyms, but unfortunately `GUID` is not one of them.

Reply:
The only reason I used GUID is that the VS Code validator give warnings for that.